utils/build-presets.ini

```diff
@@ -926,7 +926,7 @@ reconfigure
 # Use the clang that we install in the path for macros
 llvm-cmake-options=
   -DCROSS_TOOLCHAIN_FLAGS_LLVM_NATIVE='-DCMAKE_C_COMPILER=clang;-DCMAKE_CXX_COMPILER=clang++'
-  -DCLANG_DEFAULT_LINKER=gold
+  -DCLANG_DEFAULT_LINKER=ld.lld
```

Comment:
Just one note, as Evan says, the default is lld, so the better way to make this switch is to delete these 9 lines.